scripts/update/01-run.sh

```diff
@@ -58,8 +58,9 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
     fi
 
     # This makes sure systemd services are always updated (and new ones are enabled).
-    UMBREL_SYSTEMD_SERVICES="${UMBREL_ROOT}/.umbrel-${RELEASE}/scripts/umbrel-os/services/*.service"
-    for service_path in $UMBREL_SYSTEMD_SERVICES; do
+    UMBREL_SYSTEMD_SERVICES_ROOT="${UMBREL_ROOT}/.umbrel-${RELEASE}/scripts/umbrel-os/services"
+    UMBREL_SYSTEMD_SERVICES="*.service"
+    for service_path in "$UMBREL_SYSTEMD_SERVICES_ROOT"/$UMBREL_SYSTEMD_SERVICES ; do
```

Comment:
it's better to do it like this:
`"$UMBREL_SYSTEMD_SERVICES_ROOT/$UMBREL_SYSTEMD_SERVICES"` 
the whole path is enclosed in quotes.

Reply:
Can't be done.  The point of doing it the way it currently is, is that you want the shell metacharacter expansion in the `for` loop.  If I quote as per your suggestion, that won't work anymore — thus defeating the point of the `for` loop.